### ISSUE TITLE
Group Geometry - Limit search results list by group geometry

### ISSUE THREAD

**Issue by azerbini** [azerbini/eamena_v3#65]
Currently, if I am part of a group with geometric restrictions, I am still returned all the sites in the database as a list of search results, but the ones lying outside my area of access return a 403 error when clicked on. These **should not** appear in the list of results at all.

**Comment by ahaith** [comment]
Aha, I remember now. Being a member of the main edit/view group returns all resources _unless_ there is an area defined. If you set a small triangle somewhere in the sea for these, you will get the desired behaviour.

**Comment by azerbini** [comment]
@ahaith No, this doesn't seem to work. I added region-wide geometries to both edit and editplus. Then, I have a user which can only access geometries in a portion of Tunisia (and is not part of either edit or editplus, but only of a group with access to Tunisian records). The latter will still get the full list of Tunisian search results, and get 403 when clicking on sites lying outside of the geometry specified in his group.

**Comment by ahaith** [comment]
OK, will investigate on Monday. It might be useful for you to come over at some point early in the week if you have any free time?

Some good news: I have the tunisia shape you gave me successfully filtering the right resources from elasticsearch (after tweaking the kml file contents slightly).

Cheers,
Andy
________________________________

**Comment by azerbini** [comment]
Amazing news about ES! I needed this right before leaving the office :)

I have pushed some commits to admin-panel, have a look.

Have a good weekend!



Sent from my iPhone

On 10 Nov 2017, at 18:09, Andy Haith <notifications@github.com<mailto:notifications@github.com>> wrote:

OK, will investigate on Monday. It might be useful for you to come over at some point early in the week if you have any free time?

Some good news: I have the tunisia shape you gave me successfully filtering the right resources from elasticsearch (after tweaking the kml file contents slightly).

Cheers,
Andy
________________________________
